app/models/taxon.rb

```diff
@@ -13,7 +15,7 @@ class Taxon
 
   include ActiveModel::Model
 
-  validates_presence_of :title, :internal_name
+  validates_presence_of :title, :internal_name, :path_prefix, :path_slug
   validates_with CircularDependencyValidator
```

Comment:
We'll probably have to validate that the `path_prefix` is one of our allowed prefixes. The `path_slug` also needs some validation (a-z 0-9 and dashes?).